caffe2/operators/tt_linear_op.h

```diff
@@ -65,7 +65,8 @@ class TTLinearOp final : public Operator<Context> {
 
       // TODO Replace by Reshape(), once wrappers are written
       Y_buf->Resize(Y_buf->numel() / curr_rows, curr_rows);
-      Y->Resize(Y_buf->numel() / curr_rows, curr_cols);
+      Y = Output(
+          0, {Y_buf->numel() / curr_rows, curr_cols}, at::dtype<float>());
```

Comment:
Below, they use `mutable_data<T>`. Sure you want to use `float` here instead?

Reply:
this is weird, it first uses `float` but then changes to `T` later, clangr is matching the first occurrence of mutable_data so it picked `float`. I'll ping POC